### PR TITLE
data: high-prog

### PR DESCRIPTION
**Pull request by sarthkdobriyal**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5502"></a>

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
you mistakenly added the filename to the root folder please add it into the `data` folder & remember to add your exact github username. `sarthkdobriyal`

- `data/sarthkdobriyal.json` 

- you can see the official docs https://linkfree.io/docs/quickstart

**Review by priyankarpal — CHANGES_REQUESTED** [review]
- add `.json` next to your username. `data/sarthkdobriyal.json`

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/sarthkdobriyal.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "url": "https://github.com/high-prog",
+      "icon": "FaGithub"
+    },{
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/sarthk-dobriyal/",
+      "icon": "FaLinkedin"
+    },{
+      "name": "Read my blogs",
+      "url": "https://sarthk.hashnode.dev/",
+      "icon": "FaHashnode"
```
Comment (priyankarpal) [inline data/sarthkdobriyal.json:21]: `FaHashnode` is not available. Change to `SiHashnode`

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by priyankarpal — APPROVED** [review]
Awesome Looks Great 🎉

**Pull request merged**